impeller/entity/contents/content_context.cc

```diff
@@ -185,7 +179,13 @@ void ContentContextOptions::ApplyToPipelineDescriptor(
         break;
       case StencilMode::kCoverCompare:
         front_stencil.stencil_compare = CompareFunction::kNotEqual;
-        front_stencil.depth_stencil_pass = StencilOperation::kKeep;
+        front_stencil.depth_stencil_pass =
```

Comment:
I'd add a comment here that says the reference value ought to be zero.

Reply:
Ditto below.

Reply:
Added some comments here. This is also documented in the docstrings, but tried to make it a little more clear.